src/datastore.py

```diff
@@ -1,68 +1,43 @@
 from pyspark.sql import DataFrame, DataFrameWriter, DataFrameReader
-from pyspark.sql.types import *
+from pyspark.sql.functions import lit
+from schema import schema_csv, schema_meta
 
 
 class DataStore:
-    schema_csv = StructType([
-        StructField('Date', DateType()),
-        StructField('Open', DoubleType()),
-        StructField('High', DoubleType()),
-        StructField('Low', DoubleType()),
-        StructField('Close', DoubleType()),
-        StructField('Adj Close', DoubleType()),
-        StructField('Volume', LongType())
-    ])
 
-    schema_meta = StructType([
-        StructField("Nasdaq Traded", StringType()),
-        StructField("Symbol", StringType()),
-        StructField("Security Name", StringType()),
-        StructField("listing Exchange", StringType()),
-        StructField("Market Category", StringType()),
-        StructField("ETF", StringType()),
-        StructField("Round Lot Size", DoubleType()),
-        StructField("Test Issue", StringType()),
-        StructField("Financial Status", StringType()),
-        StructField("CQS Symbol", StringType()),
-        StructField("NASDAQ Symbol", StringType()),
-        StructField("NextShares", StringType())
-    ])
-
-    def __init__(self, ses, config=None):
-        self.spark = ses
+    def __init__(self, spark, config=None):
```

Comment:
I would like to introduce a component `AppContext`, which holds the config and the SparkSession. We would pass it in here as well. Please add the `AppContext` class as well. Upon instantiation, it should read the json-config (from a default path, if not specified otherwise: e.g. `project_root/config/config.json`). By passing around an `AppConfig` object, we have access to Spark and the loaded config.